ui/src/ducks/app/nodes.js

```diff
@@ -68,16 +68,16 @@ export function* fetchNodes() {
             cpu: node.status.capacity && node.status.capacity.cpu,
             control_plane:
               node.metadata &&
```

Comment:
I guess the `node.metadata && node.metadata.labels && node.metadata.labels[s] !== undefined` could be factored out :)